File: seed/php-model/trace/src/V2/Problem/VoidFunctionDefinitionThatTakesActualResult.php
<?php

namespace Seed\V2\Problem;

use Seed\Core\SerializableType;
use Seed\Core\JsonProperty;
use Seed\Core\ArrayType;
use Seed\V2\Problem\Parameter;
use Seed\V2\Problem\FunctionImplementationForMultipleLanguages;

/**
* The generated signature will include an additional param, actualResult
 */
class VoidFunctionDefinitionThatTakesActualResult extends SerializableType
{
    #[JsonProperty("additionalParameters"), ArrayType([Parameter])]
    /**
     * @var array<Parameter> $additionalParameters
     */
    public array $additionalParameters;

    #[JsonProperty("code")]
    /**
     * @var FunctionImplementationForMultipleLanguages $code
     */
    public FunctionImplementationForMultipleLanguages $code;

    /**
     * @param array<Parameter> $additionalParameters
     * @param FunctionImplementationForMultipleLanguages $code
     */
    public function __construct(
        array $additionalParameters,
        FunctionImplementationForMultipleLanguages $code,
    ) {
        $this->additionalParameters = $additionalParameters;
        $this->code = $code;
    }
}

File: seed/ruby-sdk-v2/trace/lib/seed/v_2/problem/void_function_definition_that_takes_actual_result.rb
# frozen_string_literal: true

module Seed
    module Types
        # The generated signature will include an additional param, actualResult
        class VoidFunctionDefinitionThatTakesActualResult < Internal::Types::Model
            field :additional_parameters, Internal::Types::Array[Seed::V2::Problem::Parameter], optional: false, nullable: false
            field :code, Seed::V2::Problem::FunctionImplementationForMultipleLanguages, optional: false, nullable: false

    end
end
